src/GitHubCreateMergePRs/Program.cs

```diff
@@ -1,12 +1,92 @@
-﻿using System;
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-namespace GitHubCreateMergePRs
+#r "System.Xml.Linq"
```

Comment:
I see, so here one of our .csx files is being adapted as the main file for a console app.

Reply:
Yep basically just created a project from the csx files so I could invoke it with dotnet run.